generators/app/index.js

```diff
@@ -294,15 +285,24 @@ module.exports = JhipsterAuditGenerator.extend({
     updateEntityFiles() {
       // Update existing entities to enable audit
       if (this.updateType === 'all') {
-        this.entitiesToUpdate = this.existingEntities;
+        this.auditedEntities = this.existingEntities;
       }
-      if (this.entitiesToUpdate && this.entitiesToUpdate.length > 0 && this.entitiesToUpdate !== 'none') {
-        this.log(`\n${chalk.bold.green('I\'m Updating selected entities ')}${chalk.bold.yellow(this.entitiesToUpdate)}`);
+      if (this.auditedEntities && this.auditedEntities.length > 0 && this.auditedEntities !== 'none') {
+        this.log(`\n${chalk.bold.green('I\'m Updating selected entities ')}${chalk.bold.yellow(this.auditedEntities)}`);
         this.log(`\n${chalk.bold.yellow('Make sure these classes does not extend any other class to avoid any errors during compilation.')}`);
         let jsonObj = null;
 
-        this.entitiesToUpdate.forEach((entityName) => {
-          jsonObj = this.fs.readJSON(`.jhipster/${entityName}.json`);
+        this.auditedEntities.forEach((entityName) => {
+          const entityFile = `.jhipster/${entityName}.json`;
+          jsonObj = this.fs.readJSON(entityFile);
+
+          // flag this entity as audited so the :entity subgenerator
+          // can pick up all audited entities
+          // technically this is only needed for Javers, as the custom
+          // framework obtains this list at runtime using
+          // `EntityAuditEventRepository.findAllEntityTypes`;
+          this.updateEntityConfig(entityFile, 'enableEntityAudit', true);
```

Comment:
That's the thing, I couldn't find it, in fact when I ran the main generator at first it didn't update my entity json files at all, that's why I initially thought we weren't storing the information about which entities should be audited anywhere.